package ru.third.inno.task.common.exception;

/**
 * Created by yy on 25.02.17.
 */
public class TaskDaoException extends Throwable {
}
